doc/nrf/drivers/entropy_cc310.rst

```diff
@@ -7,18 +7,19 @@ CC310 entropy driver
    :local:
    :depth: 2
 
-You can use the CC3xx entropy driver (entropy_cc310) to generate random data using the Arm CryptoCell CC310 hardware.
-This hardware is available on the nRF52840 and nRF9160 devices.
+You can use the CC3xx entropy driver (*entropy_cc310*) to generate random data using the Arm CryptoCell CC310 hardware available on the following Nordic SoCs:
 
-When used on nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+* nRF52840
+* nRF5340
+* nRF9160
 
-When used on nRF9160, the driver behavior depends on whether it is used in a secure or non-secure application:
+When used in conjunction with the nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+When used in conjunction with the nRF5340 or nRF9160, the driver behavior depends on whether it is used in a secure or non-secure application:
 
-When used on nRF5340, the driver behavior depends on whether it is used in a secure or non-secure application:
+* In a secure application, or when the :ref:`lib_spm` is not used, the entropy_cc3xx driver gathers entropy by using the CC3xx hardware through the :ref:`nrf_cc3xx_platform_readme`.
```

Comment:
Not sure about this, but should we have entropy_cc3xx between double backticks to highlight it? Or then have italics as you used that above?

Reply:
I would rather use the complete name of the library, as in the title: CC310 entropy driver.
We have some defined rules for highlighting content (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html) and it looks like the emphasis in line 10 would be fine, but in other occurrences we should just use the library name?

Also, I see inconsistency with the CC310 in the title that later becomes CC3xx... Why is this?